### ISSUE TITLE
TFException: Expects arg[0] to be string but float is provided (Unity3d)

### ISSUE THREAD

**Issue by nadir500** [migueldeicaza/TensorFlowSharp#287]
hello
i am using Speech Commands Model Exaple from TensorFlow with the variables in Unity3d like this:

```
string WAV_INPUT = "wav_data";
        string SOFTMAX_NAME = "labels_softmax";
        string[] outputScoresNames = new string[] { SOFTMAX_NAME };
```

and then add input to the model like this:

string[] labels = new string[] { "_silence_" , "_unknown_", "stop","go"};

        ```
 TextAsset model = Resources.Load("GoStop") as TextAsset;
         TFGraph  graph = new TFGraph();
         graph.Import(model.bytes);

        TFSession session = new TFSession(graph);

        var runner = session.GetRunner();

        runner.AddInput(graph[WAV_INPUT][0], audioFile);
        runner.Fetch(outputScoresNames);
        runner.Run();
```

and the error given for softmax is like that:

```
TFException: Expects arg[0] to be string but float is provided
TensorFlow.TFStatus.CheckMaybeRaise (TensorFlow.TFStatus incomingStatus, System.Boolean last) (at <1fe2de69842a4a4ba15256b83cca05f3>:0)
TensorFlow.TFSession.Run (TensorFlow.TFOutput[] inputs, TensorFlow.TFTensor[] inputValues, TensorFlow.TFOutput[] outputs, TensorFlow.TFOperation[] targetOpers, TensorFlow.TFBuffer runMetadata, TensorFlow.TFBuffer runOptions, TensorFlow.TFStatus status) (at <1fe2de69842a4a4ba15256b83cca05f3>:0)
TensorFlow.TFSession+Runner.Run (TensorFlow.TFStatus status) (at <1fe2de69842a4a4ba15256b83cca05f3>:0)
tensor.recognize (System.Single[] audioFile) (at Assets/tensor.cs:51)
tensor.Start () (at Assets/tensor.cs:23)
```

**Comment by dacostaleo** [comment]
Same problem. Did you find a solution?
